mdx2/command_line/scale.py

```
@@ -96,6 +96,7 @@ class Parameters:
     detector: DetectorModelParameters = field(default_factory=DetectorModelParameters)
     offset: OffsetModelParameters = field(default_factory=OffsetModelParameters)
     outfile: Optional[List[str]] = field(default=None, nargs="*")
```

Ruff (0.15.10)

[warning] Do not perform function call `field` in dataclass defaults

(RUF009)